src/EditorFeatures/Test/Utilities/AsynchronousOperationListenerTests.cs

```diff
@@ -79,7 +75,7 @@ public void Operation()
             using (var sleepHelper = new SleepHelper())
             {
                 var signal = new ManualResetEventSlim();
-                var listener = new Listener();
+                var listener = new AsynchronousOperationListener("test", tracking: false);
```

Comment:
Why are these all tracking: false? I don't think it's bad, it just seems odd to be called out directly.

Reply:
ah, this code is before I made them default parameter. let me remove those.